app/javascript/components/editor/TestRunOutput.tsx

```diff
@@ -1,18 +1,33 @@
 import React from 'react'
 import { TestRun } from './types'
-import { TestsGroupList } from './TestsGroupList'
+import { TestsGroupedByStatusList } from './TestsGroupedByStatusList'
+import { TestsGroupedByTaskList } from './TestsGroupedByTaskList'
 
 export const TestRunOutput = ({
   testRun,
 }: {
   testRun: TestRun
 }): JSX.Element => {
-  return testRun.version === 2 || testRun.version === 3 ? (
-    <TestsGroupList
-      tests={testRun.tests}
-      language={testRun.highlightjsLanguage}
-    />
-  ) : (
+  if (testRun.version >= 3 && testRun.tasks.length > 0) {
```

Comment:
Just as a reminder: just checking for v3 is not enough, as some exercises might not (yet) have the task mapping done.

Reply:
I've also gone with `>=` instead of `===` to allow for new versions later.